swagger.yaml

```diff
@@ -46,8 +90,10 @@ definitions:
 
   PermissionLevel:
     type: "integer"
-    minimum: 1
+    minimum: 0
```

Comment:
According to https://fa.oregonstate.edu/bic/core/core-access-and-security-structure, I didn't see the permission level 0 is defined. Did you find any permission level equals to 0 in the database?

Reply:
Yes, I found one entry where two of the permissions are level 0. Also, I found at least one entry where a level 5 permission is used for the student permissions which should only have levels 1-4.

Reply:
Interesting. I am fine with the current change for now. Could you also document what you found in the JIRA ticket and let @jaredkosanovic know?

Reply:
Sure, sounds good